h2o-docs/src/product/data-science/word2vec.rst

```diff
@@ -0,0 +1,117 @@
+Word2vec
+--------
+
+Introduction
+~~~~~~~~~~~~
+
+In text-rich datasets, the Word2vec algorithm computes vector representations of words within the data and outputs those word vectors into a single file. The vector space can include hundreds of dimensions, with each unique word in the sample (`corpus <https://en.wikipedia.org/wiki/Corpus_linguistics>`__) being assigned a corresponding vector in the space. In addition, words that share similar contexts in the corpus are placed in close proximity to one another in the space.
```

Comment:
We should probably not mention that it outputs to a "single file". The result of the algorithm is H2O's word2vec model (that can be exported as a binary model and/or as a mojo).

I like the introduction of official website of word2vec authors because it mentions the typical use of the algo (https://code.google.com/archive/p/word2vec/):

"The word2vec tool takes a text corpus as input and produces the word vectors as output. It first constructs a vocabulary from the training text data and then learns vector representation of words. The resulting word vector file can be used as features in many natural language processing and machine learning applications."